m3u8/model.py

```diff
@@ -449,6 +450,15 @@ def _create_sub_directories(self, filename):
             os.makedirs(basename, exist_ok=True)
 
 
+T = TypeVar("T")
```

Comment:
How does it work?

Reply:
It's the [Generics support that was added in 3.12](https://docs.python.org/3/library/typing.html#generics)